package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/skyux-builder-plugin-stache",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
I would prefer we create a new "release" branch after these changes get approved an in.  It's easier to backtrack through the commit/PR history if they are separated out, release from feature.

Reply:
🤷‍♀️

Reply:
🙄  LOL.  It's cleaner and easier to track specific items if a release/version bump branch is separated from the feature change.  When browsing through the commit history we can more easily identify when/where features were added/changed/fixed If they are kept under a different commit heading.  Also when tracking when a specific release went out, having that separate branch is easier to find.